code/datums/extensions/eye/_eye.dm

```diff
@@ -90,8 +90,7 @@
 			return ..(eye_extension.extension_eye)
 		if(EXTENSION_TARGET)
 			return ..(eye_extension)
-		else
```

Comment:
Not sure if this is ideal, in the case of switches the `else` section is the `default` from other languages, it's not quite the same as an unnecessary else after a return.

Reply:
I checked that in every case I removed an else in a switch, that EVERY case returned. That means that the only time the code after the switch is reached is when all conditions are false, i.e. the default case.

Reply:
That is my point. There's a distinction to be drawn between `else` as the default case of a switch and `else` as the fallthrough of a conditional and your changes are removing that visual distinction for no tangible benefit.